pkg/cephfs/volumeoptions.go

```diff
@@ -245,13 +245,13 @@ func newVolumeOptionsFromVolID(ctx context.Context, volID string, volOpt, secret
 		}
 	}
 
+	volOptions.ProvisionVolume = true
+
 	volOptions.RootPath, err = getVolumeRootPathCeph(ctx, &volOptions, cr, volumeID(vid.FsSubvolName))
 	if err != nil {
-		return nil, nil, err
+		return &volOptions, &vid, err
```

Comment:
What is the reason this is needed? I can not see it being used in this PR.

Reply:
If image is not found, we still need the options and vid returned to cleanup the journal. For reference the rbd return is similar, but we pass it the structure and refer to the same.

This is needed in DeleteVolume, when on receiving volume not found errors, we still need the contents of the passed returns to delete the journal and OMaps that are maintained.

Reply:
Ok, got it. Thanks!